integration/src/integration/java/pl/allegro/tech/hermes/integration/PublishingTest.java

```diff
@@ -291,4 +294,27 @@ public void shouldPublishInvalidJsonMessageOnValidationDryRun() {
         assertThat(response).hasStatus(CREATED);
     }
 
+    @Test
+    public void shouldPublishMessageWithTraceId() {
```

Comment:
i don't know whats the purpose of this test, the body does hardly anything with trace-id?